imageprocessor/views.py

```diff
@@ -139,3 +144,31 @@ def register(request):
         return HttpResponseRedirect(reverse('login'))
     context['form'] = form
     return render(request, 'register.html', context)
+
+class ClassifyAPI(APIView):
+
+    def post(self, request, format=None):
+        response_data = {
+            'url' : None,
+            'tags' : []
+        }
+        form = ImageForm(request.POST or None, request.FILES or None)       
+        try:
+            image_file = request.FILES['file']
+            image = Image.open(image_file)
+            tags = detect(image) 
+            if (tags):
```

Comment:
This check can never fail. The detect function throws a value error if it can't generate tags for an image, and then immediately jumps down into the valueError handler

Reply:
Oh nice I didnt realize is was no longer possible for it to return an empty string

Reply:
Will get rid  since the if is unused